src/index.html

```diff
@@ -16,17 +17,32 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form-search-bar form-search-bar--big"
+      data-qa="big"
+    >
+      <input
+        class="input-bar input-bar--big"
+        id="input-icon-big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles”"
```

Comment:
Keep your attributes correctly formatted. Since the <input> element has more than two attributes, each attribute should start on a new line with 2-space indentation. The closing bracket should align with the opening tag line.